### ISSUE TITLE
ERROR:  relation "webhooks" does not exist at character 105

### ISSUE THREAD

**Issue by typkrft** [go-vikunja/api#104]
### Description

When Opening the webhooks form from the menu in the projects view. I get a 500. In the Database logs it shows 

```logs
ERROR:  relation "webhooks" does not exist at character 105
2023-12-29 06:45:46.873 UTC [192] STATEMENT:  SELECT "id", "target_url", "events", "project_id", "secret", "created_by_id", "created", "updated" FROM "webhooks" WHERE (project_id = $1) LIMIT 50
```

Everything else seems to working properly, so I am not sure what would be causing this issue in my set. 

Compose
```yaml
services:

  vikunja-db:
    container_name: vikunja-db
    image: postgres:alpine
    environment:
      POSTGRES_USER: redacted
      POSTGRES_PASSWORD: redacted
      POSTGRES_DB: redacted
    volumes:
      - ./storage/db:/var/lib/postgresql/data
    restart: unless-stopped
    healthcheck:
      test: [ "CMD", "pg_isready", "-q", "-d", "redacted", "-U", "redacted" ]
      interval: 10s
      timeout: 5s
      retries: 3
      start_period: 60s
  

  vikunja-api:
    container_name: vikunja-api
    image: vikunja/api
    environment:
      VIKUNJA_DATABASE_HOST: redacted
      VIKUNJA_DATABASE_PASSWORD: redacted
      VIKUNJA_DATABASE_TYPE: postgres
      VIKUNJA_DATABASE_USER: redacted
      VIKUNJA_DATABASE_DATABASE: redacted
      VIKUNJA_SERVICE_JWTSECRET: redacted
      VIKUNJA_SERVICE_FRONTENDURL: "https://redacted/"
      
      VIKUNJA_REDIS_ENABLED: 1
      VIKUNJA_REDIS_HOST: 'vikunja-redis:6379'
      VIKUNJA_CACHE_ENABLED: 1
      VIKUNJA_CACHE_TYPE: redis

      PUID: 1000
      GUID: 1000
      VIKUNJA_SERVICE_TIMEZONE: US/Eastern
      VIKUNJA_DEFAULTSETTINGS_DISCOVERABLE_BY_NAME: true
      VIKUNJA_DEFAULTSETTINGS_DISCOVERABLE_BY_EMAIL: true
      VIKUNJA_SERVICE_ENABLEREGISTRATION: false

      VIKUNJA_TYPESENSE_ENABLED: true
      VIKUNJA_TYPESENSE_URL: http://vikunja-typesense:8108
      VIKUNJA_TYPESENSE_APIKEY: redacted

      VIKUNJA_MAILER_ENABLED: true
      VIKUNJA_MAILER_HOST: redacted
      VIKUNJA_MAILER_PORT: 587
      VIKUNJA_MAILER_USERNAME: redacted
      VIKUNJA_MAILER_PASSWORD: redacted
      VIKUNJA_MAILER_FROMEMAIL: redacted

      # Not currently possible to config openid through env variables
    volumes: 
      - ./storage/files:/app/vikunja/files
      - ./storage/config.yml:/app/vikunja/config.yml
    depends_on:
      vikunja-db:
        condition: service_healthy
      vikunja-redis:
        condition: service_healthy
      vikunja-typesense:
        condition: service_started
    restart: unless-stopped
    ports:
      - 8001:3456
  
  vikunja-frontend:
    container_name: vikunja-frontend
    image: vikunja/frontend
    restart: unless-stopped
    # environment:
    #   VIKUNJA_API_URL: "https://redacted/api/v1"
    ports:
      - 8002:80

  vikunja-redis:
    image: redis
    container_name: vikunja-redis
    healthcheck:
      test: ["CMD", "redis-cli", "ping"]
      interval: 10s
      timeout: 30s
      retries: 3

  vikunja-typesense:
    container_name: vikunja-typesense
    image: typesense/typesense:0.26.0.rc42
    restart: unless-stopped
    ports:
      - "8108:8108"
    volumes:
      - ./storage/typesense-data:/data
    command: '--data-dir /data --api-key=redacted --enable-cors'
```

Both current "latest" images for front end and backend. 

### Vikunja Frontend Version

sha256:298816ba5b44db814bba5a45f19a8ef6a9d52ac9826daf14fc5d5853dcef9cb6

### Vikunja API Version

sha256:84bde8580e03cf0b41853597850d64471f8a43e5c1b46cecda95a0424b227d7c

### Browser and version

FF 122.0b3

### Can you reproduce the bug on the Vikunja demo site?

No

### Screenshots

<img width="840" alt="Screenshot 2023-12-29 at 02 01 26" src="https://github.com/go-vikunja/api/assets/12100960/8dff7699-0974-4d7a-a6c0-ff79edb29898">

**Comment by kolaente** [comment]
Looks like this issue: https://kolaente.dev/vikunja/api/issues/1655

**Comment by typkrft** [comment]
I should have been looking at votes. My bad. I'll take a look at the thread thanks. I'm not sure if it matters or not but this is a brand new setup not a migration which I saw mentioned. But I'll continue reading through.

**Comment by typkrft** [comment]
I am consistently reproducing this with my compose. I didn't see a resolution either, or should I try deleting the migration and restarting the API. Nothing to do with trying to migrate.

**Comment by kolaente** [comment]
Looks like the table was not created when starting from scratch, only when upgrading from a previous version. Hence I didn't see this error with any of my own installations.

Should be fixed in 09696aec1bea647a5bfc7be16b31054626d721e4. Please check with the next unstable build.

**Comment by typkrft** [comment]
Confirmed resolved in that setup. Aprecciated
